packages/tools/pie-icons-webc/README.md

```diff
@@ -0,0 +1,113 @@
+
+# pie-icons-webc
+
+Shared PIE Icon Components for Web Components.
+
+This package generates an Web Component iconset for applications using the base [pie-icons](https://www.npmjs.com/package/@justeattakeaway/pie-icons) package. The SVGs in pie-icons are compiled into single file components that can be imported into applications.
+
+---
+
+[![npm version](https://img.shields.io/npm/v/@justeattakeaway/pie-icons-webc.svg)](https://img.shields.io/npm/v/@justeattakeaway/pie-icons-webc.svg)
+
+---
+## Usage
+
+### Installation
+
+Add the module to your project
+
+```bash
+yarn add @justeattakeaway/pie-icons-webc
+```
+
+### Applications
+
```

Comment:
Could you add an example of how an icon can be pulled into a Lit Web Component as well?